.bazel-bin

```diff
@@ -0,0 +1 @@
+/home/olemarkus/.cache/bazel/_bazel_olemarkus/ca218e60a15b6f2ea61e6bebee2077bb/execroot/__main__/bazel-out/k8-fastbuild/bin
```

Comment:
Not quite sure how these were picked up in the cherry-pick.

Reply:
because they are not in .gitignore in that branch ... thanks